ui/src/pages/backoffice/DocumentRequest/DocumentRequestDetails/components/DocumentRequestActions/RejectAction.js

```diff
@@ -47,7 +47,7 @@ export default class RejectAction extends React.Component {
     }
   };
 
-  render() {
+  renderDropdown() {
```

Comment:
I would put generating options in separate method, and pass them to dropdown as a prop (and the dropdown itself in the render function) - IMO it is better readable, but it might be in the next PR if you find it relevant